### ISSUE TITLE
Upgrade issue 19.0.10.1 -> 19.0.11.1

### ISSUE THREAD

**Issue by apw1388** [nextcloud/server#27037]
### Steps to reproduce
1. `.occ upgrade` from existing 19.0.10.1 installation

### Expected behaviour
Upgrade should work without errors

### Actual behaviour
```
Checking for update of app workflowengine in appstore
Checked for update of app "workflowengine" in appstore 
An unhandled exception has been thrown:
Error: Call to undefined method OC\Server::get() in /var/www/nextcloud/lib/private/Repair.php:162
Stack trace:
#0 /var/www/nextcloud/lib/private/Updater.php(273): OC\Repair::getRepairSteps()
#1 /var/www/nextcloud/lib/private/Updater.php(130): OC\Updater->doUpgrade('19.0.11.1', '19.0.10.1')
#2 /var/www/nextcloud/core/Command/Upgrade.php(255): OC\Updater->upgrade()
#3 /var/www/nextcloud/3rdparty/symfony/console/Command/Command.php(255): OC\Core\Command\Upgrade->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /var/www/nextcloud/3rdparty/symfony/console/Application.php(1012): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /var/www/nextcloud/3rdparty/symfony/console/Application.php(272): Symfony\Component\Console\Application->doRunCommand(Object(OC\Core\Command\Upgrade), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 /var/www/nextcloud/3rdparty/symfony/console/Application.php(148): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 /var/www/nextcloud/lib/private/Console/Application.php(215): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 /var/www/nextcloud/console.php(100): OC\Console\Application->run()
#9 /var/www/nextcloud/occ(11): require_once('/var/www/nextcl...')
#10 {main}
```

### Server configuration

**Operating system:** CentOS7

**Web server:** Apache 2.4.6

**Database:** MariaDB: mysql  Ver 15.1 Distrib 5.5.64-MariaDB, for Linux (x86_64) using readline 5.1

**PHP version:** 7.2.27

**Nextcloud version:** 19.0.11.1

**Updated from an older Nextcloud/ownCloud or fresh install:** update from 19.0.10.1

**Where did you install Nextcloud from:** Download zip file from Nexcloud download server

**List of activated apps:**
<details>
<summary>App list</summary>
Enabled:
  - accessibility: 1.5.0
  - activity: 2.12.1
  - admin_audit: 1.9.0
  - announcementcenter: 3.8.1
  - b2sharebridge: 2.0.0
  - calendar: 2.0.5
  - checksum: 0.4.5
  - circles: 0.19.13
  - cloud_federation_api: 1.2.0
  - comments: 1.9.0
  - contacts: 3.5.1
  - contactsinteraction: 1.0.0
  - data_request: 1.6.0
  - dav: 1.15.0
  - deck: 1.2.7
  - drawio: 1.0.0
  - federatedfilesharing: 1.9.0
  - federation: 1.9.0
  - files: 1.14.0
  - files_markdown: 2.3.2
  - files_pdfviewer: 1.8.0
  - files_rightclick: 0.16.0
  - files_sharing: 1.11.0
  - files_trashbin: 1.9.0
  - files_versions: 1.12.0
  - files_videoplayer: 1.8.0
  - firstrunwizard: 2.8.0
  - group_everyone: 0.1.7
  - groupfolders: 7.1.2
  - logreader: 2.4.0
  - lookup_server_connector: 1.7.0
  - lrswitchboardbridge: 0.4
  - nextcloud_announcements: 1.8.0
  - notifications: 2.7.0
  - oauth2: 1.7.0
  - onlyoffice: 6.4.2
  - password_policy: 1.9.1
  - photos: 1.1.0
  - privacy: 1.3.0
  - provisioning_api: 1.9.0
  - quota_warning: 1.8.0
  - recommendations: 0.7.0
  - serverinfo: 1.9.0
  - settings: 1.1.0
  - sharebymail: 1.9.0
  - support: 1.2.1
  - survey_client: 1.7.0
  - systemtags: 1.9.0
  - tasks: 0.13.6
  - text: 3.0.1
  - theming: 1.10.0
  - twofactor_backupcodes: 1.8.0
  - updatenotification: 1.9.0
  - user_saml: 3.2.3
  - viewer: 1.3.0
  - workflowengine: 2.1.0
Disabled:
  - encryption
  - files_external
  - user_ldap
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by michaelwimmer** [comment]
To add: there seems to be a related question (without answers) in the help forum: https://help.nextcloud.com/t/upgrade-18-0-14-to-19-0-11-fails/115570

**Comment by chrda81** [comment]
I did the upgrade from 19.0.5.2 to 19.0.11.1 some minutes ago and ran into the same problem. I looked at the file nextcloud/lib/private/Repair.php at line 162 and find the call `\OC::$server->get(LookupServerSendCheck::class)`. The _\OC::$server->get_ call seems to be introduced into newer server versions because it references the LookupServerSendCheck method, which seems to be from version 22. If you comment the line 162 in the file nextcloud/lib/private/Repair.php, the update succeeds.

**Comment by MorrisJobke** [comment]
Let us stop the 19.0.11 rollout and then we fix this.

**Comment by thomasBrecho** [comment]
same problem here update from 19.0.10 to 19.0.11 on ubuntu 18.04 LTS PHP 7.3

**Comment by MorrisJobke** [comment]
Fixed with #27041 . We plan to do the 19.0.12 with only this fix today.
